vision/getting-started/image_segmentation.ipynb

```diff
@@ -0,0 +1,1003 @@
+        "* Use the confidence_threshold and max_predictions parameters to filter and limit results\n",
+        "* You can request multiple items be detected by separating them with commas. Hundreds of classes can be set in a single prompt."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "unrHSqhqHmlI"
+      },
+      "outputs": [],
+      "source": [
+        "mode = \"prompt\"\n",
+        "prompt = \"green watermelon, cantaloupe, price tag\"\n",
+        "confidence_threshold = 0.1\n",
+        "max_predictions = None\n",
+        "\n",
+        "response = call_vertex_image_segmentation(\n",
+        "    image_bytes=ENCODED_IMAGE_BYTES,\n",
+        "    gcs_uri=None,\n",
+        "    mime_type=None,\n",
+        "    mode=mode,\n",
+        "    prompt=prompt,\n",
+        "    scribble_bytes=None,\n",
+        "    mask_dilation=None,\n",
+        "    max_predictions=max_predictions,\n",
+        "    confidence_threshold=confidence_threshold,\n",
+        ")\n",
+        "\n",
+        "print(f\"Number of predictions is {str(len(response.predictions))}\")\n",
+        "\n",
+        "bbox_image = draw_bounding_boxes(INPUT_IMAGE_PIL, response)\n",
+        "overlayed_image = overlay(INPUT_IMAGE_PIL, response)\n",
+        "display_horizontally([INPUT_IMAGE_PIL, bbox_image, overlayed_image], figsize=(25, 25))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yQXy4qaOJwte"
+      },
+      "source": [
+        "### Interactive segment mode\n",
+        "\n",
+        "Interactive segmentation enables point-to-mask or scribble-to-mask generation.\n",
+        "This mode is perfect for application developers.\n",
+        "\n",
+        "1. Set the brush size\n",
+        "2. Run the canvas cell below to draw a mask to guide the interactive segmentation model.\n",
+        "3. Click and hold your mouse over the image and drag it to draw a mask, then let go. The scribble is saved automatically.\n",
+        "  * You can draw a single point to \"select\" an object to segment\n",
+        "  * A rough line over any object works as well\n",
+        "  * You can try scribbling over more of the desired object to see how the generated mask changes.\n",
+        "  * You can draw one or multiple scribbles.\n",
+        "5. When you're finished drawing, run the Segmentation cell"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "cellView": "form",
+        "id": "7MAipuW1Klrc"
+      },
+      "outputs": [],
+      "source": [
+        "# @title [Canvas] Draw a mask scribble\n",
+        "# @markdown Run this cell to begin drawing. Re-run this cell to start over.\n",
+        "brush_size = 15  # @param {type:\"number\"}\n",
+        "\n",
+        "import base64\n",
+        "import re\n",
+        "\n",
+        "from google.colab import html\n",
```

Comment:
Use a different library other than colab to do this so that it can work in Vertex AI Workbench or other tools.

Also, if you need to use HTML, I'd recommend putting that in a separate file and loading it in instead of putting it all in a python string.

Reply:
Removed this section - I'll implement a Gradio UI for it instead.